sources/packages/api/src/route-controllers/program-info-request/program-info-request.institutions.controller.ts

```diff
@@ -284,16 +275,16 @@ export class ProgramInfoRequestController extends BaseController {
   }
 
   /**
-   * Get all application of a location in an institution
+   * Get all applications of a location in an institution
    * with Program Info Request (PIR) status completed and required
    * @param locationId location that is completing the PIR.
-   * @returns student application list of an institution location
+   * @returns student application list of an institution location.
    */
   @HasLocationAccess("locationId")
   @Get(":locationId/program-info-request")
   async getPIRSummary(
-    @Param("locationId") locationId: number,
-  ): Promise<PIRSummaryDTO[]> {
+    @Param("locationId", ParseIntPipe) locationId: number,
+  ): Promise<PIRSummaryAPIOutDTO[]> {
```

Comment:
a qn, are we doing date refactoring as part of the story, `studyStartPeriod: getISODateOnlyString(offering?.studyStartDate),
        studyEndPeriod: getISODateOnlyString(offering?.studyEndDate),`

Reply:
Date-only values should be sent from the API in ISO format.